tests/Webserver.API.UnitTests/ApiBulkTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) 2021, Siemens AG
+//
+// SPDX-License-Identifier: MIT
+using NUnit.Framework;
+using RichardSzalay.MockHttp;
+using Siemens.Simatic.S7.Webserver.API.Models.Requests;
+using Siemens.Simatic.S7.Webserver.API.Services.RequestHandling;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Webserver.API.UnitTests
+{
+    class ApiBulkTests : Base
+    {
+        [Test]
+        public void ApiBulk_RequestsSameId()

```

Comment:
With unit test functions it would be nice to add also the expected result to the name, like: 
```suggestion
        public void ApiBulk_RequestsSameId_ThrowsArgumentException()

```

Reply:
applied for many unit tests